lib/components/hh5p/player/index.js

```diff
@@ -28,6 +28,7 @@ var jsx_runtime_1 = require("react/jsx-runtime");
 var react_1 = require("react");
```

Comment:
remove that file, whole `lib` dir, 
Lets not VCS complied files 
This is just for `npm publish`